spyder/plugins/updatemanager/workers.py

```diff
@@ -193,25 +287,31 @@ class WorkerUpdate(BaseWorker):
     def __init__(self, stable_only):
         super().__init__()
         self.stable_only = stable_only
-        self.latest_release = None
-        self.update_available = False
+        self.asset_info = None
         self.error = None
         self.checkbox = False
         self.channel = None
 
-    def _check_update_available(
-        self,
-        releases: list[Version],
-        github: bool = True
-    ):
-        """Checks if there is an update available from releases."""
+    def _check_asset_available(self, release: None | Version):
+        """Checks if there is an asset available for release."""
```

Comment:
```suggestion
        """Check if there is an asset available for release."""
```

Docstrings are written in imperative language.